### DISCUSSION TITLE
httpstatus for tweet

### DISCUSSION THREAD

**Discussion by clashen70** [noweh/twitter-api-v2-php#45] [Q&A] [answered]
Thanks for great API!
I am using it solely to post tweets.
I have one question.
How do i get the httpstatus from Twitter after tweeting to know if the tweet got published?

_$response = $client->tweet()->create()->performRequest(['text' => 'Testing API V2 ']);_

**Comment by noweh** [comment, marked as answer]
The variable $response is always an HTTP 200 response. Indeed, in case of an error, the API will return an Exception containing the error message and the relevant HTTP code.
You can see this process here: https://github.com/noweh/twitter-api-v2-php/blob/master/src/AbstractController.php in the function performRequest.
